packages/hint/package.json

```diff
@@ -16,19 +16,23 @@
     "caniuse-api": "^3.0.0",
     "chalk": "^2.4.2",
     "content-type": "^1.0.4",
+    "css-select": "^2.0.2",
     "debug": "^4.1.1",
     "eventemitter2": "^5.0.1",
     "file-type": "^10.9.0",
     "file-url": "^2.0.2",
     "globby": "^9.1.0",
     "is-ci": "^2.0.0",
     "is-svg": "^4.0.0",
+    "jsdom": "^13.2.0",
```

Comment:
Do we need `jsdom` in here? I don't see it being used in `hint`.

Reply:
Used by `packages/hint/src/lib/utils/dom/create-jsdom.ts`, but it appears after various refactors this is only consumed by `connector-local`. Probably best to move the code there and remove the dependency from `hint`.

Reply:
we still have `create-jsdom.ts` in utils/dom. But it is only used in `connector-local` so I think I'm going to remove `create-jsdom.ts` from utils.